frontend/src/old-pages/Clusters/util.tsx

```diff
@@ -27,3 +28,23 @@ export function selectCluster(clusterName: any, navigate: any)
   if(oldClusterName !== clusterName)
     setState(['app', 'clusters', 'selected'], name);
 }
+
+export function getScripts(customActions: [key: string]) {
+  const scriptName = (script: string) => {
+    let suffix = script.slice(script.lastIndexOf('/') + 1);
+    return suffix.slice(0, suffix.lastIndexOf('.'));
+  }
+  
+  let allScripts = [];
+  for(let actionName of ['OnNodeStart', 'OnNodeConfigured'])
+  {
+    if(getIn(customActions, [actionName, 'Script']))
+      allScripts.push(scriptName(getIn(customActions, [actionName, 'Script'])));
+    for(let arg of (getIn(customActions, [actionName, 'Args']) || []))
+    {
+      if(arg.length > 0 && arg[0] !== '-')
+        allScripts.push(scriptName(arg));
+    }
+  }
+  return allScripts
+}
```

Comment:
This would a be a great opportunity to write a test for this function! Feel free to ping for help if needed!